frontend/src/components/views/Graph.svelte

```diff
@@ -567,20 +566,18 @@
 	});
 </script>
 
-<LayoutRow
-	class="graph"
+<div
+	class=".layout-row graph"
```

Comment:
Shouldn't be a `.` here. Also I'd prefer to not have a DIV mimic a component (`LayoutRow`) which it isn't, so I'd prefer just adding the same CSS properties that `.layout-row` gets to the CSS for `.graph` and that way it can avoid pretending to be a `layout-row`.